source/devices/AM335X/linux/Release_Specific_Release_Notes.rst

```diff
@@ -179,114 +179,11 @@ Yocto
-   SITSW-1376,"am335x-evm,am335x-sk","Matrix demo does not start until XFRM netlink socket initialization is complete","Reload localhost URL after XFRM netlink socket initialization is completed"
-
-|
-
-.. _release-specific-u-boot-known-issues:
-
-U-Boot Known Issues
--------------------
-.. csv-table::
-   :header: "Record ID","Platform", "Title","Workaround"
-   :widths: 15, 30, 70, 30
-
-   "LCPD-37017","am335x-evm","u-boot NAND boot failure with SDK 8.2 on AM335x EVM",""
-   "LCPD-19133","am335x-evm,am335x-hsevm,am335x-ice,am335x-sk","Netconsole output corrupted when CONFIG_NETCONSOLE_BUFFER_SIZE >= 281",""
-
-
-|
-
-.. _release-specific-linux-kernel-known-issues:
-
-Linux Kernel Known Issues
--------------------------
-.. csv-table::
-   :header: "Record ID", "Platform", "Title", "Workaround"
-   :widths: 5, 10, 70, 10
-
-   "LCPD-27472","am335x-evm","DRM stress failed ,Test setting different video modes frequently 1000 times",""
-   "LCPD-25272","am335x-evm","Crypto_M_PERF_openssl_perf_software test Fails",""
-   "LCPD-24728","am335x-evm","Power measurement with Standby/Suspend/Resume failure",""
-   "LCPD-24456","am335x-evm,am335x-hsevm,am335x-ice,am335x-sk","Move IPC validation source from github to git.ti.com",""
-   "LCPD-24302","am335x-evm","The speed test built in to OpenSSL - cryptographic",""
-   "LCPD-24182","am335x-evm,beaglebone-black","Powermgr_xs_func_simple_suspend/standby resume tests",""
-   "LCPD-24104","am335x-evm","nandtest util crashes NAND ubifs filesystem",""
-   "LCPD-23096","am335x-ice","PTP broken with PRU-ETH on AM335x ICEv2",""
-   "LCPD-22417","am335x-evm","No throughput on ipsec aes128 hardware test",""
-   "LCPD-17790","am335x-evm","AM335x: USB Device: 15% performance drop",""
-   "LCPD-17673","am335x-evm,beaglebone-black","No software documentation for the Timer module",""
-   "LCPD-17449","am335x-evm,am335x-hsevm,am335x-ice,am335x-sk,beaglebone-black","libasan_preinit.o is missing in devkit",""
-   "LCPD-14855","am335x-evm,am335x-ice,am335x-sk","omap_i2c_prepare_recovery() function can Lock System",""
-   "LCPD-13947","am335x-evm","nativesdk-opkg is broken in the devkit",""
-   "LCPD-10221","am335x-evm","Longer resume times observed on setup with usb device cable",""
-   "LCPD-9455","am335x-evm","Kernel Warning reported for a USB audio device when listing with pulseaudio",""
-   "LCPD-9372","am335x-evm","Nand stress tests failed on a particular am335x-evm board",""
-   "LCPD-8636","am335x-evm","Serial corruption being seen in kernel",""
-   "LCPD-8133","am335x-evm","USB: ""cannot reset"" errors observed sometimes",""
-   "LCPD-7955","am335x-evm","Uncorrectable Bitflip errors seen after switch to SystemD","Workaround to erase the NAND flash completely if flashed with an incompatible flash writer. SystemD tries to mount all partitions and that is the reason this is being seen now."
-   "LCPD-5362","am335x-evm","MUSB: Isoch IN only utilises 50% bandwidth",""
+.. note::
```

Comment:
I don't think this should be a note if it's the expected location for the release bug report moving forward. It should just be a paragraph about where they can find this information and why. For example:


```rst
To unify software issue tracking, release specific issue tracking is currently
handled through the `Software Incident Report (SIR) portal
<https://sir.ext.ti.com/>`.
```

Reply:
Offline sync: This is not the permanent solution yet. We're fine with the note until a proper solution is determined.